hw/ip/rom_ctrl/data/rom_ctrl_testplan.hjson

```diff
@@ -74,8 +76,10 @@
     {
       name: rom_ctrl_check_cg
       desc: '''
-            Collect coverage on the outputs sent to the power manager to confirm
+            - Collect coverage on the outputs sent to the power manager to confirm
             that we see pass and fail resutls.
+            - Collect coverage to ensure that d_valid doesn't go high when rom check 
+            is in progerss.
```

Comment:
We could be a bit more specific here. Presumably the scoreboard or some assertion will check the property we need. So the coverage point should instead be checking that we saw `a_valid` at some point when the ROM check was in progress. (Also, I think you actually want to check that `a_ready` doesn't go high)